frontend/src/components/subscribers/index.tsx

```diff
@@ -206,11 +206,15 @@ export const Subscribers = () => {
               ),
             }}
           >
-            {(labels || []).map((label) => (
-              <MenuItem key={label.id} value={label.id}>
-                {label.name}
-              </MenuItem>
-            ))}
+            {!!labels.length ? (
+              labels.map((label) => (
+                <MenuItem key={label.id} value={label.id}>
+                  {label.name}
+                </MenuItem>
+              ))
+            ) : (
+              <MenuItem disabled>No label found</MenuItem>
```

Comment:
Hi @JaiSharma1710, can you please use i18n to support French and English translations.
Kindly find bellow the paths to the i18n translation files :
French translations : `frontend/src/i18n/fr/translation.json`
English translations : `frontend/src/i18n/en/translation.json`
I suggestion to use the template feature to generate generic translations